toolkit/tools/internal/packagerepo/repomanager/rpmrepomanager/rpmrepomanager.go

```diff
@@ -16,12 +16,31 @@ import (
 
 // CreateRepo will create an RPM repository at repoDir
 func CreateRepo(repoDir string) (err error) {
+	logger.Log.Debugf("Creating RPM repository in (%s)", repoDir)
+
+	err = CleanupRepo(repoDir)
+
+	if err != nil {
+		// This error is not fatal.
+		logger.Log.Debugf("Failed to cleanup repo %s: %v", repoDir, err)
```

Comment:
Why do we say it's not fatal?

Reply:
It's not fatal if the error is file not found - that simply means that the repo didn't exist beforehand. I will check for this error only.

Reply:
Got it, now I see that we no longer do the `!os.IsNotExist(err)` check for `os.Remove`, so I guess that's what you have in mind.

Reply:
Since I couldn't find a standard way to handle this error, I reverted the changes.